py/desisurveyops/status_zhist.py

```diff
@@ -114,18 +114,23 @@ def process_zhist(
 
         # AR select the tiles
         sel = obs_progs == program
-        sel &= np.in1d(obs_tiles, e["TILEID"])
-        if npassmax is not None:
-            t = Table.read(fns["ops"]["tiles"])
-            t = t[t["PASS"] < npassmax]
-            sel &= np.in1d(obs_tiles, t["TILEID"])
-
-        if npassmax is not None:
-            log.info(
-                "found {} tiles with PROGRAM = {} and PASS < {}".format(
-                    sel.sum(), program, npassmax
-                )
-            )
+        sel &= np.isin(obs_tiles, e["TILEID"])
+        # if npassmax is not None:
+        #     t = Table.read(out_fns["ops"]["tiles"])
+        #     t = t[t["PASS"] < npassmax]
+        #     sel &= np.isin(obs_tiles, t["TILEID"])
+
+        # DG For any skip cases that get passed, e.g. skipping pass 5 in BRIGHT1B
+
+        if skip_pass is not None:
+            log.warning(f"For PROGRAM={program_str} ignoring PASS={skip_pass}")
+            fn = out_fns["ops"]["tiles"]
```

Comment:
This (re) definition of `fn` was not in the code that previous version of the code, can you explain what this is doing and why it has been added?